tensorflow/core/util/gpu_device_functions.h

```diff
@@ -82,6 +84,10 @@ class GpuGridRange {
   T end_;
 };
 
+#ifndef TENSORFLOW_USE_ROCM
+template <typename... T>
+using CudaGridRange = GpuGridRange<T...>;
```

Comment:
Potentially I can create another macro for this:

> #define CREATE_CUDA_TEMPLATE_TYPE_ALIAS(type, cuda_alias) \
      template <typename... Args> \
      using cuda_alias = type<Args<&&...>; 

But considering this is a one-time pattern, just skipped it.